src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/NewLineAnalyzer.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Azure.ClientSdk.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class NewLineAnalyzer: DiagnosticAnalyzer
```

Comment:
NewLineAnalyzer is checking for  trailing whitespace as well as new lines ... is there a name that would encompass both?